src/test/java/duke/test/ParserTest.java

```diff
@@ -0,0 +1,56 @@
+package duke.test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import duke.Parser;
+import duke.commands.AddCommand;
+import duke.commands.Command;
+import duke.commands.DeleteCommand;
+import duke.commands.ListCommand;
+import duke.commands.MarkCommand;
+import duke.commands.QuitCommand;
+import duke.commands.UnmarkCommand;
```

Comment:
Perhaps could just use 

`duke.command.*`

if all the package files in the commands are needed